### PR TITLE
[DRAFT] Exclude rw blobs from exported models

### PR DESCRIPTION
**Pull request by mryzhov**
### Details:
 - QF to test import/export models w/o RW memory allocation
### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/plugins/intel_gna/gna_model_serial.cpp [hunk 1]
```diff
@@ -528,7 +530,7 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
     }
 
     // once structure has been written lets push gna graph
-    os.write(reinterpret_cast<char*>(basePointer), gnaGraphSize);
+    os.write(reinterpret_cast<char*>(basePointer), ro_size);
```
Comment (dmitriikhurtin) [inline src/plugins/intel_gna/gna_model_serial.cpp:533]: rw_size?

Comment (mryzhov) [reply src/plugins/intel_gna/gna_model_serial.cpp:533]: No, this is right, gnaGraphSize = ro_size + rw_size, we removed rw region at all.

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**